packages/utils/package.json

```diff
@@ -42,6 +42,7 @@
 	"dependencies": {
 		"@directus/constants": "workspace:*",
 		"@directus/storage": "workspace:*",
+		"bowser": "^2.11.0",
```

Comment:
Could we maybe switch that to [ua-parser-js](https://www.npmjs.com/package/ua-parser-js)? This library is actively maintained, while it doesn't seem to be the case for `bowser`.

Reply:
We could but there would be no benefit from using a more recently maintained version of a parser as the underlying logic is likely the same and bowser still has a ton of downloads.